### PR TITLE
Fix 2023 courts

### PR DESCRIPTION
**Pull request by flooie**
Fix 9 broken courts.

afcca
bap9
bap10
ca4
ca7
ed_louisiana
mspb_p
mspb_u
tax

### CODE REVIEW

File: tests/examples/opinions/united_states/afcca_example.compare.json [hunk 1]
```diff
@@ -1,332 +1,32 @@
...
-    "download_urls": "tests/examples/opinions/united_states/afcca_opinions/cp/roblero-38874.u.pdf",
+    "case_dates": "2023-01-11",
+    "case_names": "United States v. Goldsmith",
+    "download_urls": "tests/examples/opinions/united_states/afcca_opinions/cp/goldsmith_-_40148_u_1066513.pdf",
     "precedential_statuses": "Unpublished",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "ACM 38874",
-    "case_name_shorts": "Roblero"
+    "docket_numbers": "40148",
```
Comment (mlissner) [inline tests/examples/opinions/united_states/afcca_example.compare.json:9]: Hm, big diff in docket number format here, no?

File: tests/examples/opinions/united_states/ca7_example.html [hunk 2]
```diff
@@ -1,336 +1,391 @@
...
+<description>The latest opinions released by the US 7th Circuit Court of Appeals</description>
+<language>en-us</language>
+<pubDate>
+Tue, 20 Feb 2007 10:05:00 CDT
+</pubDate>
+<lastBuildDate>
+Fri, 17 Sep 2021 11:36:00 CST
+</lastBuildDate>
+
+<item>
```
Comment (mlissner) [inline tests/examples/opinions/united_states/ca7_example.html:21]: Let's trim this file so it's much shorter (and thus faster). Maybe only five entries, or something?

File: tests/examples/opinions/united_states/tax_example.compare.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   {
     "case_dates": "2021-12-08",
     "case_names": "Alice J. Coggin",
-    "download_urls": "https://public-api-blue.dawson.ustaxcourt.gov/public-api/21580-19/919cb4ce-fb26-4cb3-a0f3-a1a2f0e6927b/public-document-download-url",
+    "download_urls": "https://public-api-green.dawson.ustaxcourt.gov/public-api/21580-19/919cb4ce-fb26-4cb3-a0f3-a1a2f0e6927b/public-document-download-url",
```
Comment (mlissner) [inline tests/examples/opinions/united_states/tax_example.compare.json:5]: Hm, this looks like they're doing [green/blue deployments](https://docs.aws.amazon.com/whitepapers/latest/overview-deployment-options/bluegreen-deployments.html), and makes me wonder if it'll swap back to blue next time they release code. I hope not, but it's possible. I guess we'll see.

**Review by mlissner — CHANGES_REQUESTED** [review]
A few tweaks, please, but looks good.

File: tests/examples/opinions/united_states/tax_example.compare.json (hunk 3 above)
Comment (flooie) [reply tests/examples/opinions/united_states/tax_example.compare.json:5]: I saw the blue green deployment, and assumed that blue was from all the test and green was the actual production.  When we built this, it seemed very much to be in development

File: tests/examples/opinions/united_states/afcca_example.compare.json (hunk 1 above)
Comment (flooie) [reply tests/examples/opinions/united_states/afcca_example.compare.json:9]: yes, I saw that, but the docket numbers look like they include ACM to me.

Comment (flooie) [reply tests/examples/opinions/united_states/afcca_example.compare.json:9]: but, on the website, I think its less clear. 
Screenshot 2023-01-19 at 10 41 16 AM

Comment (mlissner) [reply tests/examples/opinions/united_states/afcca_example.compare.json:9]: Both seem acceptable. So be it.

File: tests/examples/opinions/united_states/ca7_example.html (hunk 2 above)
Comment (flooie) [reply tests/examples/opinions/united_states/ca7_example.html:21]: √

File: tests/examples/opinions/united_states/tax_example.compare.json (hunk 3 above)
Comment (mlissner) [reply tests/examples/opinions/united_states/tax_example.compare.json:5]: We shall see!

**Comment by flooie** [comment]
> Feel free to merge when ready.

 I am unable to merge this.  Or override it.

**Pull request merged**